src/test/resources/dokumentasjon/features/hovedregler/arbeidsforhold/regelflyt_regel_21.feature

```diff
@@ -24,28 +24,27 @@ Egenskap: Regelflyt for regel 21
 
   Scenariomal: Regelflyt regel 21: Bruker får "uavklart" på spørsmålet om medlemskap hvis arbeidsperioden ikke dekker referanse perioden
     Gitt  følgende arbeidsforhold fra AAReg
-      | Fra og med dato    | Til og med dato    | Arbeidsgivertype | Arbeidsforholdstype | Arbeidsgiver Id |
-      | <Første fra og med>| <Første til og med>| Organisasjon     | NORMALT             | 1               |
-      | <Andre fra og med> | <Andre til og med> | Organisasjon     | NORMALT             | 1               |
+      | Fra og med dato     | Til og med dato     | Arbeidsgivertype | Arbeidsforholdstype | Arbeidsgiver Id |
+      | <Første fra og med> | <Første til og med> | Organisasjon     | NORMALT             | 1               |
 
     Når medlemskap beregnes med følgende parametre
       | Fra og med dato | Til og med dato | Har hatt arbeid utenfor Norge |
-      |   10.06.2020    | 10.06.2020      | Nei                           |
+      | 10.06.2020      | 10.06.2020      | Nei                           |
 
     Så skal svaret være "<Medlemskap>"
     Og skal regel "21" gi svaret "<Svar>"
 
     Eksempler:
-      | Første fra og med | Første til og med | Andre fra og med | Andre til og med | Svar | Medlemskap |
-      | 01.01.2020        |                   | 01.01.2020       |                  | Ja   |      Ja    |
-      | 01.01.2020        | 01.06.2020        | 01.06.2020       |                  | Ja   |      Ja    |
-      | 01.01.2020        | 01.06.2020        | 04.06.2020       |                  | Nei  |   UAVKLART |
+      | Første fra og med | Første til og med | Svar | Medlemskap |
+      | 01.01.2018        |                   | Ja   | Ja         |
+      | 01.01.2018        | 11.06.2020        | Ja   | Ja         |
```

Comment:
hvorfor må du endre dato?

Reply:
Testen tok ikke hensyn til regel 3 (sjekk arbeidsforhold 1 år tilbake i tid) siden denne ble overstyrt. Nå som overstyringen blir slått av måtte datoene endres for at testene skulle fungere